code/drasil-example/Drasil/Projectile/IMods.hs

```diff
@@ -33,17 +33,18 @@ import Drasil.Projectile.Unitals (flightDur, landPos, launAngle, launSpeed,
 iMods :: [InstanceModel]
 iMods = [timeIM, landPosIM, offsetIM, messageIM]
```

Comment:
Adding to the below comment, we also need to use the "getEqModQdsFromIMs" function on this in the Body.hs -- exact name is on the other PR.